### PR TITLE
Repurpose Repo

### PR DESCRIPTION
**Pull request by mistermoe**

### CODE REVIEW

File: test-harness/README.md
Comment (decentralgabe) [inline test-harness/README.md:1]: rename

**Comment by mistermoe** [comment]
@finn-tbd and others. let me know if moving the current test harness into the `test-harness` directory works

File: README.md
Comment (decentralgabe) [inline README.md:200]: I think this is fine to start but would like to see these checks/x's dynamically generated

**Review by decentralgabe — APPROVED** [review]

File: README.md
Comment (decentralgabe) [reply README.md:200]: and need to align this with the current test harness

Comment (finn-tbd) [reply README.md:200]: that's certainly one of the things that's in the works. We have an HTML report like this now, and from there generating a markdown report shouldn't be too hard. Currently blocked by lack of any functioning tests.

**Comment by finn-tbd** [comment]
all the  files that come from the tbd template repo should probably remain in the root (`CODEOWNERS`, `CODE_OF_CONDUCT.md`, `CONTRIBUTING.md`, `LICENSE`, `GOVERNANCE.md`) as should `action.yml` (must be in the root to use this repo as a gh action in other repos). Otherwise this seems fine.

**Comment by mistermoe** [comment]
@finn-tbd makes sense. done https://github.com/TBD54566975/sdk-development/pull/10/commits/a94cabdf3b518f84be8278800ab1ed3185511224

File: README.md
Comment (mistermoe) [reply README.md:200]: agreed. the purpose for adding these tables here manually is to make it easier to figure out what issues need to be created for each respective SDK. the intent is to remove them as mentioned here:

image

**Pull request merged**
